content/blog/2023-10-23-introducing-localstack-desktop-application-for-local-cloud-development-testing/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Introducing LocalStack Desktop Application for local cloud development & testing
+description: Get started with the new Desktop Application to seamlessly manage your LocalStack container while developing, testing, and debugging your cloud & serverless applications.
+lead: Get started with the new Desktop Application to seamlessly manage your LocalStack container while developing, testing, and debugging your cloud & serverless applications.
+date: 2023-10-23
+lastmod: 2023-10-23
+images: []
+contributors: ["LocalStack Team"]
+tags: ['news']
+show_cta_1: true
+---
+
+We are excited to announce that we have released an all-new Desktop Application to make cloud development with LocalStack a breeze! The Desktop application underscores our unwavering commitment to make LocalStack accessible for our users, who would like to use a local cloud sandbox for development, testing, and experimentation, and truly embrace the inner dev loop. The Desktop Application now enables users to control, manage, and visualize over 80+ emulated AWS cloud services, running entirely on your local machine.
+
+{{< img-simple src="localstack-desktop-resource-browser.png" width=300 alt="Image of LocalStack Desktop Application showcasing the local AWS Resource Browsers">}}
+
+## What's new?
+
+Last year, we released [LocalStack Cockpit](https://localstack.cloud/blog/2022-02-07-localstack-cockpit/) — Our first iteration towards building a fully integrated desktop client to control, access, and manage LocalStack with a click of a button. We received a [lot of feedback](https://github.com/localstack/cockpit/issues) from the community during the beta phase, and after spending months working on it, we formulated a new product for our users to use LocalStack more efficiently. 
+
+To provide our users with the very best, we've made the decision to deprecate LocalStack Cockpit for new users. Going forward, we will help migrate our existing users and get our new users started with to the LocalStack Desktop Application.
+
+{{< img-simple src="localstack-desktop-containers.png" width=300 alt="Image of LocalStack Desktop Application showcasing the running containers">}}
+
+The new LocalStack Desktop Application features an improved user experience which is in line with our Web Application experience, with [local Resource Browsers](https://docs.localstack.cloud/user-guide/web-application/resource-browser/) available for 30+ AWS services, and an intuitive way to manage your LocalStack container and view LocalStack logs through an easy-to-use interface. 
+
+The LocalStack Desktop Application is available to download via an [executable from the Web Application]() or on the [Apple Store]() and [Windows Store](). You can download the application for free and get started!
```

Comment:
Actually, we have not planned to release it to the Apple store (yet), so I would remove it...also you can already put links (but wait before merging this) since on the web application it will live at https://app.localstack.cloud/download and for the windows store the link is https://apps.microsoft.com/detail/9NTRNFT9ZWS2